pulp_ansible/app/urls.py

```diff
@@ -225,8 +228,12 @@
 v3_plugin_urls = [
     # path:var captures /, so it has to have something at the end to make it work
     # correctly.
-    path("content/<path:distro_base_path>/collections/", include(v3_collection_urls)),
-    path("content/<path:distro_base_path>/namespaces/", include(namespace_urls)),
+    path(
+        "content/<path:distro_base_path>/collections/index/<str:namespace>/<str:name>/",
+        include(v3_collection_detail_urls),
+    ),
```

Comment:
So we give the regex more to munch on? I wonder if you could still craft a collection name to break this... But we don't allow "/" there, right?

Reply:
The "/" character should be an invalid one for a collection namespace or name because it wouldn't result in a valid python package name. I can add another test to validate that the import process fails on such a thing though.

Reply:
I think, it's ok.